packages/manager/apps/procedures/src/data/hooks/rgdp/useRGDP.tsx

```diff
@@ -0,0 +1,46 @@
+import { useMutation } from '@tanstack/react-query';
+import {
+  sendForm,
+  UploadLink,
+  getUploadDocumentsLinks,
+  FormValue,
+} from '@/data/api/rgdp/rgdpApi';
+
+export const useRGDPUploadLinks = ({
```

Comment:
Reading the doc from SIA, the form data will be sent in this call, so you need to update these two functions